docs/index.adoc

```diff
@@ -11999,12 +11999,17 @@ class TestOutput {
 
 === Testing the Exit Code
 Testing the exit code of applications that call `System.exit` can be tricky but is not impossible.
+Depending on your programming language, your environment and your testing framemork, you have several options: 
 
-==== Java 8+ and `catchSystemExit`
-Applications that use JUnit 5 or Java 8+ can use Stephan Birkner's https://github.com/stefanbirkner/system-lambda[System-Lambda] project for testing the https://github.com/stefanbirkner/system-lambda#systemexit[exit code].
-For example:
+Java 8:: You can use Stephan Birkner's https://github.com/stefanbirkner/system-lambda[System-Lambda] project for testing the https://github.com/stefanbirkner/system-lambda#systemexit[exit code]
+Java/Kotlin + Junit 5:: You can use Todd Ginsberg's https://github.com/tginsberg/junit5-system-exit[System.exit()] extension for Junit 5. The project README mentions common https://github.com/tginsberg/junit5-system-exit#use-cases[use cases].
+Java + Junit 4:: You can use Stephan Birkner's https://stefanbirkner.github.io/system-rules[System-Rules] project for testing the https://stefanbirkner.github.io/system-rules/#ExpectedSystemExit[exit code].
+Kotlin + Kotest:: Kotlin coders you may use the https://kotest.io/[Kotest] test framework which comes with https://kotest.io/docs/extensions/system_extensions.html#system-exit-extensions[System Exit extensions] that allow you to test the exit code.
```

Comment:
I have not seen the convention to use two colon characters (`Java 8::`, `Java/Kotlin + Junit 5::`, etc) before. Is this an AsciiDoc thing that I am not aware of maybe?

Would it make sense to make this a bullet point list? That makes it easy to digest for readers, I think.

> * Java 8: You can use ...
> * Java/Kotlin + Junit 5: You can use...
> * Java + Junit 4: You can use...
> * Kotlin + Kotest: Kotlin coders you may use...

Reply:
>  Is this an AsciiDoc thing that I am not aware of maybe?

Yes, it's markup for [description lists](https://docs.asciidoctor.org/asciidoc/latest/lists/description/).

> Would it make sense to make this a bullet point list?

I liked the markup, but a bullet list is pretty much equivalent. I changed it to a bullet list.